src/index.js

```diff
@@ -60,11 +60,12 @@ class LinesEllipsis extends React.PureComponent {
     mirrorProps.forEach((key) => {
       this.canvas.style[key] = targetStyle[key]
     })
+    this.canvas.style.width = this.target.clientWidth + 'px'
```

Comment:
What's the difference with `mirrorProps.width`?

Reply:
If width is something like "100%", mirrored width is "100%" too.
But original component will use width of it's parent, and mirrored - of <body>.
Result - different width

Reply:
Mirrored - of `body` tag*

Reply:
AFAIK `window.getComputedStyle` returns pixel value for `width`.

Reply:
I've made some tests and `window.getComputedStyle` returns width in pixels. But yesterday I really saw `width: 100%` in `this.canvas` style property. 

By the way, pull request was only about fixing `basedOn` property - last two commits got here by mistake.

Reply:
So please make this pull request focus and remove the last two commit.